salt/minion.py

```diff
@@ -3205,6 +3205,29 @@ def _post_master_init(self, master):
             self.schedule.delete_job(master_event(type='alive', master=self.opts['master']), persist=True)
             self.schedule.delete_job(master_event(type='failback'), persist=True)
 
+        # proxy keepalive
+        proxy_alive_fn = fq_proxyname+'.alive'
```

Comment:
Yeah... bizarrely seems so. I tried to follow the code format from above, e.g. `fq_proxyname+'.init'` -- but I can change if requested.